android/build.gradle

```diff
@@ -63,25 +62,9 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation 'androidx.annotation:annotation:1.2.0'
 
-    // TODO: un-comment the api line and remove the implementation once purchases-android is released
-    // api "com.revenuecat.purchases:purchases:$purchases_version"
```

Comment:
maybe we can document how this works somewhere in notion? ideally it wouldn't be pushed again but i could see it being useful for local development